tests/base/test_objective_function.py

```diff
@@ -352,7 +352,7 @@ def test_mul(self, unpack_on_add):
         combo_mul = 3.5 * combo
         assert len(combo_mul) == 1
         assert combo_mul.multipliers == [3.5]
-        assert combo_mul.objfcts == [combo]
+        # assert combo_mul.objfcts == [combo]
```

Comment:
We can not do straight equal anymore since the function gets copied and we end up with two different instances (not the same id)